From 830857ddebab11d1a4e957621cf939dda3d41637 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Sat, 5 Jan 2019 11:51:55 +0100
Subject: [PATCH 29/30] Don't explicitly build uneeded packages

* libemoji doesn't exist
* libprotobuf-cpp-full is always built and part of VNDK
* librsjni is always built by renderscript

Change-Id: I379aa405f937d54fc7ab1fb7a120683e91f2bbac
---
 config/common.mk | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index ac08ddde..24ca9e38 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -140,16 +140,10 @@ PRODUCT_PACKAGES += \
 
 # Optional packages
 PRODUCT_PACKAGES += \
-    libemoji \
     LiveWallpapersPicker \
     PhotoTable \
     Terminal
 
-# Include explicitly to work around GMS issues
-PRODUCT_PACKAGES += \
-    libprotobuf-cpp-full \
-    librsjni
-
 # Custom Lineage packages
 PRODUCT_PACKAGES += \
     AudioFX \
-- 
2.17.1

